migrations/sql/afterMigrate__1__seed_data.sql

```diff
@@ -1765,3 +1765,5 @@ UPDATE document_template SET form_spec_json = '[
       "read-only": true
     }
 ]' WHERE document_template_code = 'ESL';
+
+UPDATE permit_amendment SET is_generated_in_core = true WHERE permit_conditions_last_updated_date IS NOT NULL;
```

Comment:
This update statement should be in the other SQL file. We only want it to run once, and this file is specifically for populating the seed data.